src/pint/observatory/__init__.py

```diff
@@ -110,11 +110,24 @@ def _add_aliases(cls, obs, aliases):
                     obs_aliases.append(alias)
             o._aliases = obs_aliases
 
+    @classmethod
+    def clear_registry(cls):
```

Comment:
As long as the registry is populated on import, this is rather problematic as the user doesn't have an obvious way to repopulate the registry (using `importlib` to force a reload is decdedly non-obvious, particularly as you have to reload several modules). Maybe to build a test fixture it's better to just save the dictionary `cls._registry` and restore it?

Reply:
but it isn't populated only on import.  the `read_observatories()` function re-populates it.